python/pip_install/repositories.bzl

```diff
@@ -3,6 +3,9 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 
+# Avoid a load from @bazel_skylib repository as users don't necessarily have it installed
```

Comment:
Looks like versions.bzl tries to load skylib anyway. https://github.com/bazelbuild/rules_python/pull/533/files#diff-86c5bb17276af4fdb85612635eabfdaf9b91676ae6bac37ee78a09947dfd6045R1

Reply:
thanks for fixing it!